src/components/ha-cover-tilt-controls.ts

```diff
@@ -0,0 +1,118 @@
+import { HassEntity } from "home-assistant-js-websocket";
+import {
+  LitElement,
+  property,
+  internalProperty,
+  CSSResult,
+  css,
+  customElement,
+  TemplateResult,
+  html,
+} from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
+
+import { UNAVAILABLE } from "../data/entity";
+import { HomeAssistant } from "../types";
+import CoverEntity from "../util/cover-model";
+
+import "./ha-icon-button";
+
+@customElement("ha-cover-tilt-controls")
+class HaCoverTiltControls extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @internalProperty() private _stateObj!: HassEntity;
+
+  private _entityObj?: CoverEntity;
+
+  public set stateObj(stateObj: HassEntity) {
+    this._entityObj = new CoverEntity(this.hass, stateObj);
+    this._stateObj = stateObj;
+  }
```

Comment:
A `set` doesn't check if it is changed, better to do it in `updated` (we then also don't need the `_stateObj`)